msi/GitProduct.wxs

```diff
@@ -115,15 +124,15 @@
     <Feature Id='Git'>
       <Component Directory='INSTALLFOLDER'>
         <RegistryValue Root='HKLM' Key='SOFTWARE\GitForWindows' Name='InstallFolder' Value='[INSTALLFOLDER]' Type='string' />
-        <Environment Name='PATH' Value='[BinFolder]' Action='set' Part='last' System='yes' />
```

Comment:
I'll add something along those lines to the commit message. That's a good suggestion. Thanks!